Upstream: https://github.com/nmeum/android-tools/commit/d4285173e3d1a64a8d262e769e1017e419ec0b83
From d4285173e3d1a64a8d262e769e1017e419ec0b83 Mon Sep 17 00:00:00 2001
From: meator <meator.dev@gmail.com>
Date: Fri, 14 Aug 2026 12:52:54 +0200
Subject: [PATCH] cmake: Remove useless cpp from target

This file pull in nonstandard header sys/cdefs.h header, which is not
provided by default on musl.
---
 vendor/CMakeLists.libbase.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vendor/CMakeLists.libbase.txt b/vendor/CMakeLists.libbase.txt
index 08e07064733b..92103523af0a 100644
--- a/vendor/CMakeLists.libbase.txt
+++ b/vendor/CMakeLists.libbase.txt
@@ -14,7 +14,6 @@ add_library(libbase STATIC
     libbase/stringprintf.cpp
     libbase/strings.cpp
     libbase/threads.cpp
-    libbase/test_utils.cpp
     libbase/errors_unix.cpp)
 
 target_include_directories(libbase PUBLIC
-- 
2.55.0

